app/cafe.py

```diff
@@ -0,0 +1,18 @@
+from app.errors import (NotVaccinatedError,
+                        OutdatedVaccineError,
+                        NotWearingMaskError)
+import datetime
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if not "vaccine" in visitor:
+            raise NotVaccinatedError
+        elif visitor["vaccine"]["expiration_date"] < datetime.date.today():
+            raise OutdatedVaccineError

```

Comment:
The `OutdatedVaccineError` should be raised with a descriptive message. Consider using a message like: `raise OutdatedVaccineError('Vaccine is expired.')`.